portfolio/src/main/webapp/index.html

```diff
@@ -33,15 +33,7 @@
       <!-- Should change to url later -->
       <label for="video-id-2">Id&nbsp;</label>
       <input type="text" id="video-id-2" name="videoId"><br>
-      <label for="title-2">Title&nbsp;</label>
-      <input type="text" id="title-2" name="title"><br>
-      <label for="description-2">Description</label><br>
-      <textarea id="description-2" name="description" rows="4" cols="50"></textarea><br>
-      <label for="thumbnailUrl-2">ThumbNail Url&nbsp;</label>
-      <input type="text" id="thumbnailUrl-2" name="thumbnailUrl"><br>
-      <label for="isPublic-2">Public?&nbsp;</label>
-      <input type="checkbox" id="isPublic-2" name="isPublic"><br>
-      <input type="submit" value="Submit">
+      <input type="submit" value="Analyze Video">
```

Comment:
there shouldn't be a need for the user to re-input the videoId since we already have the key as a URL Parameter by the time we are ready to analyze

Reply:
Good point! I was planning to do this in my next PR since it will change how things work significantly (the modular request system will allow the page to be updated without reloading). I might not do it here for now since it'll have to be rewritten anyway.